api/SVGGeometryElement.json

```diff
@@ -247,90 +423,122 @@
+              }
+            ],
+            "chrome_android": [
+              {
+                "version_added": "57"
+              },
+              {
+                "version_added": "18",
+                "version_removed": "57",
+                "partial_implementation": true,
+                "notes": "Only supported for <a href='https://developer.mozilla.org/docs/Web/API/SVGPathElement'><code>SVGPathElement</code></a>, not all <code>SVGGeometryElement</code> objects."
+              }
+            ],
             "edge": {
-              "version_added": null
-            },
-            "firefox": {
-              "version_added": "61"
-            },
-            "firefox_android": {
-              "version_added": "61"
-            },
+              "version_added": "79"
+            },
+            "firefox": [
+              {
+                "version_added": "61"
+              },
+              {
+                "version_added": "1.5",
+                "version_removed": "61",
+                "partial_implementation": true,
+                "notes": "Only supported for <a href='https://developer.mozilla.org/docs/Web/API/SVGPathElement'><code>SVGPathElement</code></a>, not all <code>SVGGeometryElement</code> objects."
+              }
+            ],
+            "firefox_android": [
+              {
+                "version_added": "61"
+              },
+              {
+                "version_added": "4",
+                "version_removed": "61",
+                "partial_implementation": true,
+                "notes": "Only supported for <a href='https://developer.mozilla.org/docs/Web/API/SVGPathElement'><code>SVGPathElement</code></a>, not all <code>SVGGeometryElement</code> objects."
+              }
+            ],
             "ie": {
               "version_added": false
             },
-            "opera": {
-              "version_added": null
-            },
-            "opera_android": {
-              "version_added": null
-            },
-            "safari": {
-              "version_added": null
-            },
-            "safari_ios": {
-              "version_added": null
-            },
-            "samsunginternet_android": {
-              "version_added": null
-            },
-            "webview_android": {
-              "version_added": null
-            }
+            "opera": [
+              {
+                "version_added": "44"
+              },
+              {
+                "version_added": "≤12.1",
+                "version_removed": "44",
+                "partial_implementation": true,
+                "notes": "Only supported for <a href='https://developer.mozilla.org/docs/Web/API/SVGPathElement'><code>SVGPathElement</code></a>, not all <code>SVGGeometryElement</code> objects."
+              }
+            ],
+            "opera_android": [
+              {
+                "version_added": "43"
```

Comment:
This is not a mistake. The off-by-13 rule held up to Opera 42 on both desktop and mobile, then things got out of sync. Opera desktop 43 was Blink 56, but Android 43 skipped to 59.